runtime/Java/src/org/antlr/v4/runtime/tree/IterativeParseTreeWalker.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * [The "BSD license"]
+ *  Copyright (c) 2012 Terence Parr
```

Comment:
Isn't that copyright wrong? Neither exists this file since 2012 nor was it written by Sam + Ter.